app/scripts/proactive/templates/file-browser-template.html

```diff
@@ -28,6 +28,7 @@ <h5>
                 <a id="delete-file-btn" href="javascript:void(0)" class="pointer pa-tool-btn" title="Delete the selected file or folder">
                     <i class="fas fa-trash"></i>
                 </a>
+                <label style="padding-left: 160px">Filter: <input type="text" id="filter-files"/> </label>
```

Comment:
can you add a title attribute to the label "Filter files according to a glob pattern"